extensions/color-picker/src/extract-color.tsx

```diff
@@ -0,0 +1,114 @@
+import { Action, ActionPanel, getSelectedFinderItems, Grid, Icon, showToast, Toast } from "@raycast/api";
+import { useEffect, useState } from "react";
+import { extractColor } from "swift:../swift/extract-color";
+
+export interface FinalColor {
```

Comment:
Can we change it to `type`? `interface` is mergeable by other same interfaces. This would run the risk of being merged by mistake.